app/models/katello/concerns/hostgroup_extensions.rb

```diff
@@ -76,5 +76,5 @@ def inherited_ancestry_attribute(attribute)
 end
 
 class ::Hostgroup::Jail < Safemode::Jail
-  allow :content_source, :rhsm_organization_label
+  allow :content_source, :rhsm_organization_label, :subscription_manager_configuration_url
```

Comment:
Would be nice to maybe open a redmine issue against that. ACK from me though if you want to merge this PR.